### PR TITLE
Confirmation email for certified warden applications

### PR DESCRIPTION
**Pull request by captainmangoC4**
Send confirmation email for certified warden applications and update success message

### CODE REVIEW

File: functions/apply-for-certified-contributor.ts [hunk 1]
```diff
@@ -111,17 +117,84 @@ async function handler(event) {
...
+      if (err) {
+        reject({
+          statusCode: err.status || 500,
+          body: JSON.stringify({ error: err.message }),
+        });
+      }
+
+      const dataToSend = {
+        to: recipients,
+        message: message.toString("ascii"),
```
Comment (joshuashort) [inline functions/apply-for-certified-contributor.ts:180]: utf-8 instead?

File: functions/apply-for-certified-contributor.ts [hunk 2]
```diff
@@ -111,17 +117,84 @@ async function handler(event) {
...
+        }
+        resolve({
+          statusCode: 201,
+          body: "Your request has been submitted",
+        });
+      });
+    });
+  });
+
+  return await emailPromise;
```
Comment (joshuashort) [inline functions/apply-for-certified-contributor.ts:197]: Shouldn't return this?

**Comment by joshuashort** [comment]
Maybe do a separate linting PR at some point -- it adds a lot of noise to look through.

**Review by joshuashort — APPROVED** [review]

File: functions/apply-for-certified-contributor.ts (hunk 2 above)
Comment (captainmangoC4) [reply functions/apply-for-certified-contributor.ts:197]: Yes, I did mean to return that. The netlify lambda functions have to return an object with statusCode and body. The promise resolves with the object, so I return the resolved promise. But correct me if there's something I'm misunderstanding here.

Comment (joshuashort) [reply functions/apply-for-certified-contributor.ts:197]: I see. Maybe it'd make more sense to explicitly return a success object, at the end of everything.

Comment (captainmangoC4) [reply functions/apply-for-certified-contributor.ts:197]: Ah, yeah that makes sense.

**Review by joshuashort — APPROVED** [review]

**Pull request merged**